examples/next-login/src/pages/home/signin.tsx

```diff
@@ -0,0 +1,207 @@
+import { Text, Stack, Alert } from '@ultraviolet/ui'
+import { Icon } from '@ultraviolet/icons'
+import {
+  Form,
+  TextInputFieldV2,
+  TextInputField,
+  Submit,
+  useForm,
+  RadioField,
+  DateField,
+  FormErrors,
+} from '@ultraviolet/form'
+import { Theme, css, useTheme } from '@emotion/react'
+import { useState } from 'react'
+
+type FormValues = {
+  email: string
+  password: string
+  gender: string
+  firstname: string
+  lastname: string
+  birthdate: Date
+}
+
+const emailRegex = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
+const mockErrors: FormErrors = {
+  maxDate: ({ maxDate }) => `Date must be lower than ${maxDate?.toString()}`,
+  maxLength: ({ maxLength }) =>
+    `This field should have a length lower than ${maxLength}`,
+  minDate: ({ minDate }) => `Date must be greater than ${minDate?.toString()}`,
+  minLength: ({ minLength }) =>
+    `This field should have a length greater than ${minLength}`,
+  pattern: () => `This field should match the regex`,
+  required: () => 'This field is required',
+  max: ({ max }) => `This field is too high (maximum is : ${max})`,
+  min: ({ min }) => `This field is too low (minimum is: ${min})`,
+}
+
+const bodyStyle = (theme: Theme) => css`
+  .form-box {
+    margin: 5% 30% 5% 30%;
+    background: ${theme.colors.secondary.background};
+    padding: 2%;
+  }
+
+  .icon {
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .info-text {
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .age-input {
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+  }
+`
+
+const SignIn = () => {
+  const theme = useTheme()
+  const methods = useForm({
+    mode: 'onChange',
+    defaultValues: {
+      email: '',
+      password: '',
+      gender: 'mr',
+      firstname: '',
+      lastname: '',
+      birthdate: new Date(),
+    },
+  })
+  const [values, setValues] = useState(methods.watch())
+  const [tooYoung, setTooYoung] = useState(false)
+  const [alertSubmit, setAlertSubmit] = useState(
+    <Text as="div" variant="body">
+      {' '}
+    </Text>,
+  )
```

Comment:
It's not recommended to put a full component into a useState as it is made for storing data state it wouldn't be efficient and it hasn't been built for this usage. It should be use to condition a render or store text.